### ISSUE TITLE
add options to set user id and group id of spawned commands

### ISSUE THREAD

**Issue by dirk-zimoch** [ralphlange/procServ#17]
Options -u / --uid and -g / --gid to set user and group id of the spawned command would be nice. Should accept user/group names as well as numeric ids.

**Comment by bfrk** [comment]
Feature creep, if you ask me. Why not run procServ itself as user `ioc` like this: `runuser -u ioc -- procServ ...`?

**Comment by anjohnson** [comment]
The difference is that procServ itself is then running as that user, so the IOC itself or another process started by it could interfere with procServ. You could run just the ioc under runuser though with `procServ ... -- runuser -u ioc ...`

**Comment by bfrk** [comment]
True, it's not exactly the same thing. OTOH, running procServ as root isn't ideal either. And I like it when the log files and the communication sockets belong to the same user that runs the IOC. YMMV.
